app/cmd/rpc/server.go

```diff
@@ -2,17 +2,18 @@ package rpc
 
 import (
 	"encoding/json"
-	"github.com/julienschmidt/httprouter"
+	"fmt"
 	"io"
 	"io/ioutil"
 	"log"
 	"net/http"
-)
 
-const (
-	APIVersion = "0.2.0"
```

Comment:
what happens to releases, do we change the app.Tag value to "" each time?

Reply:
We change the version as needed, and the tag on a git tag basis

Reply:
what do you think about merging app.Tag and app.Version in 1 variable?

Reply:
the usage of `app.Tag` allows for fine grained behavior for whenever we release `stable` tags which should be separate of `RC`